packages/wallet/ui/src/components/ProvisionDialog.jsx

```diff
@@ -24,9 +26,52 @@ const errors = {
 // TODO: Read this from the chain via rpc.
 const CREATION_FEE = '10 BLD';
 
-const ProvisionDialog = ({ onClose, open, address, href, keplrConnection }) => {
+// 100 IST
+const MINIMUM_PROVISION_POOL_BALANCE = 100_000_000n;
```

Comment:
consider,
```suggestion
const MINIMUM_PROVISION_POOL_BALANCE = 100 * 1_000_000n;
```
to call separate the "100"

Reply:
Done